thunder/__init__.py

```diff
@@ -1014,6 +1014,16 @@ def last_compile_options(fn: Callable, /) -> None:
         print(f"\t{option}")
 
 
+def print_auto_registered_torch_ops(fn: Callable, /) -> set[str] | None:
+    """Returns a set of auto-registered Torch operator names present in the given JIT-compiled function."""
+    op_names = set()
+    trc = last_traces(fn)[0]
+    for bsym in trc.bound_symbols:
+        if getattr(bsym.sym.meta, "__name__", None) == "meta_func":
```

Comment:
Instead of checking the name, can we add an attribute, tag, label, etc. to bound symbols that are auto-registered that can be queried, instead? That seems more robust and clear